### PR TITLE
paperno

### LINKED ISSUE #683: Lexical Questionnaires by Paperno and Ryzhova
online at: https://sites.google.com/site/denispaperno/papers/questionnaires.zip

I did not yet fully understand the idea, but they use adjectives and word vectors to check for context that is useful to elicit a certain meaning.

Link to full text is here: https://scholarspace.manoa.hawaii.edu/handle/10125/24857

The list is not long. Probably interesting in the context of semantic norm data.

### PR DESCRIPTION
**Pull request by Schweikhard**
fixes #683

I turned the data into table format.
I'm not sure about some parts of my mapping - could someone speaking Russian take a look at it please? They cases in question mainly consist of cases where different words could be mapped to the same concept.

The automatically created json file mentions a column "gloss". I removed it from that file, or why is it added by default?

### CODE REVIEW

File: concepticondata/conceptlists/Paperno-2019-99.tsv [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Paperno-2019-99-35	35	tixij	quiet	location properties	1314	QUIET
+Paperno-2019-99-36	36	poslednij	last	order	1529	LAST (FINAL)
+Paperno-2019-99-37	37	sledujuščij	next	order	1354	NEXT
+Paperno-2019-99-38	38	spokojnyj	calm	personality		
+Paperno-2019-99-39	39	uverennyj	confident	personality		
+Paperno-2019-99-40	40	veselyj	funny	personality	1976	MERRY
+Paperno-2019-99-41	41	bol'šoj	big	size	1202	BIG
+Paperno-2019-99-42	42	krupnyj	large	size	1202	BIG
+Paperno-2019-99-43	43	nebol'šoj	small	size	1246	SMALL
+Paperno-2019-99-44	44	ogromnyj	huge	size	1202	BIG
```
Comment (LinguList) [inline concepticondata/conceptlists/Paperno-2019-99.tsv]: I'd leave this unliked: ogromnyj is rather "HUGE", so it is not only big, but bigger than big.

File: concepticondata/conceptlists/Paperno-2019-99.tsv [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Paperno-2019-99-45	45	malen'kij	little	size	1246	SMALL
+Paperno-2019-99-46	46	glubokij	deep	depth	1593	DEEP
+Paperno-2019-99-47	47	melkij	shallow	depth	193	SHALLOW
+Paperno-2019-99-48	48	nizkij	low	height	2191	LOW (HEIGHT)
+Paperno-2019-99-49	49	vysokij	high	height	1265	HIGH
+Paperno-2019-99-50	50	dlinnyj	long	length	1203	LONG
+Paperno-2019-99-51	51	korotkij	short	length	1645	SHORT
+Paperno-2019-99-52	52	širokij	wide	width	1243	WIDE
+Paperno-2019-99-53	53	tolstyj	thick	width	1244	THICK
+Paperno-2019-99-54	54	tonkij	thin	width	2249	THIN (OF SHAPE OF OBJECT)
```
Comment (LinguList) [inline concepticondata/conceptlists/Paperno-2019-99.tsv]: tonkij is used for both persons to indicate they are thin and for objects, so I would say: use the braoder THIN.

File: concepticondata/conceptlists/Paperno-2019-99.tsv [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+Paperno-2019-99-55	55	čelovečeskij	human	social value	683	PERSON
+Paperno-2019-99-56	56	strannyj	strange	social value	600	STRANGE
+Paperno-2019-99-57	57	detskij	childish	social value		
+Paperno-2019-99-58	58	osobyj	special	social value		
+Paperno-2019-99-59	59	svobod- nyj	free	social value	2316	FREE
```
Comment (LinguList) [inline concepticondata/conceptlists/Paperno-2019-99.tsv]: should be "svobodnyj"

File: concepticondata/conceptlists/Paperno-2019-99.tsv [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+Paperno-2019-99-60	60	glavnyj	main	social value	1078	MAIN
+Paperno-2019-99-61	61	važnyj	important	social value	1285	IMPORTANT
+Paperno-2019-99-62	62	interesnyj	interesting	social value		
+Paperno-2019-99-63	63	prostoj	simple	social value	686	EASY
```
Comment (LinguList) [inline concepticondata/conceptlists/Paperno-2019-99.tsv]: simple is not easy, unlink.

File: concepticondata/conceptlists/Paperno-2019-99.tsv [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+Paperno-2019-99-66	66	russkij	Russian	social value		
+Paperno-2019-99-67	67	voennyj	military	social value		
+Paperno-2019-99-68	68	ser’eznyj	serious	social value		
+Paperno-2019-99-69	69	živoj	living	social value	1422	BE ALIVE
+Paperno-2019-99-70	70	lučšij	best	social value		
+Paperno-2019-99-71	71	sobstvennyj	own	social value		
+Paperno-2019-99-72	72	ženskij	feminine	social value		
+Paperno-2019-99-73	73	obščij	common	social value		
+Paperno-2019-99-74	74	sovetskij	Soviet	social value		
+Paperno-2019-99-75	75	bystryj	quick	speed	1631	FAST
```
Comment (LinguList) [inline concepticondata/conceptlists/Paperno-2019-99.tsv]: bystryj and skoryj differ in meaning: one means fast in form of a movement, and one means fast in form of time succession (first aid is "skoryj", i.e., the help that comes in short time after wards, not the help that runs fast). I'd say: map the more proto-typical one, maybe bystryj, and leave the other one unmapped.

File: concepticondata/conceptlists/Paperno-2019-99.tsv [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+Paperno-2019-99-83	83	rannij	early	time	672	EARLY
+Paperno-2019-99-84	84	dolgij	long	time	418	FOR A LONG TIME
+Paperno-2019-99-85	85	legkij	light	weight	1052	LIGHT (WEIGHT)
+Paperno-2019-99-86	86	tjaželyj	heavy	weight	1210	HEAVY
+Paperno-2019-99-87	87	čistyj	clean	other	704	CLEAN
+Paperno-2019-99-88	88	polnyj	full	other	1429	FULL
+Paperno-2019-99-89	89	celyj	whole	other	2277	WHOLE
+Paperno-2019-99-90	90	golyj	naked	other	590	NAKED
+Paperno-2019-99-91	91	pustoj	empty	other	1624	EMPTY
+Paperno-2019-99-92	92	železnyj	iron	other	621	IRON
```
Comment (LinguList) [inline concepticondata/conceptlists/Paperno-2019-99.tsv]: this is an adjective, so it is "made from iron"

**Comment by LinguList** [comment]
again another nice dataset. with the few changes, plus the resolution of conflicts, this is ready to be merged!

**Comment by Schweikhard** [comment]
Thanks, I corrected the mistakes!

There still are some things mapped to the same concept: nebol'šoj and malen'kij both to SMALL and dobryj and milyj both to NICE - is that okay?

> this is an adjective, so it is "made from iron"

Ah, I had meant to leave it unmapped, thanks for spotting this!

**Comment by LinguList** [comment]
> There still are some things mapped to the same concept: nebol'šoj and malen'kij both to SMALL and dobryj and milyj both to NICE - is that okay?

malenkij is small, dobryj is nice, the others should be unlinked.

**Comment by Schweikhard** [comment]
Thanks!
Ah, I meant KIND, not NICE, but shouldn't make a difference. Corrected that and gonna merge.

**Pull request merged**